rosidl_typesupport_c/src/type_support_dispatch.hpp

```diff
@@ -75,6 +76,7 @@ get_typesupport_handle_function(
       auto clib = static_cast<const Poco::SharedLibrary *>(map->data[i]);
       lib = const_cast<Poco::SharedLibrary *>(clib);
       if (!lib->hasSymbol(map->symbol_name[i])) {
+        fprintf(stderr, "Failed to find symbol '%s' in library\n", map->symbol_name[i]);
```

Comment:
The `library_name` is not available in this scope since it might be a repeated invocation. I also don't think its going to be a likely case (finding the library but doesn't contain the symbol) so I didn't bother to reconstruct the library name.